src/utils/SortPeople.ts

```diff
@@ -0,0 +1,43 @@
+import { Person } from '../types';
+
+interface SortPeopleOptions {
+  sort: string | null;
+  order: string | null;
+}
+
+export const sortPeople = (
```

Comment:
file name should be in camelCase for such things
PascalCase for files with React components